gusto/advection.py

```diff
@@ -191,18 +192,30 @@ def apply(self, x_in, x_out):
 
 class EmbeddedDGAdvection(DGAdvection):
 
-    def __init__(self, state, Vdg, continuity):
+    def __init__(self, state, V, Vdg=None, continuity=False):
```

Comment:
You've changed the constructor signature, but haven't updated the embedded dg advection tests, which still use the old signature.